app/(dashboard)/layout.tsx

```diff
@@ -17,6 +14,24 @@ const Logo = () => (
   </span>
 );
 
+const GitHubButton = async () => {
```

Comment:
Make the button have the same look & feel (size, shape) as the `View on GitHub` button from the homepage.